Packs/MicrosoftGraphAPI/Integrations/MicrosoftGraphAPI/MicrosoftGraphAPI.py

```diff
@@ -86,7 +86,7 @@ def test_command(client: MsGraphClient) -> CommandResults:  # pragma: no cover
     return CommandResults(readable_output='```✅ Success!```')
 
 
-def generic_command(client: MsGraphClient, args: Dict[str, Any]) -> CommandResults:
+def generic_command(client: MsGraphClient, args: Dict[str, Any]) -> Union[CommandResults, str]:
```

Comment:
No need for str return here see my next comment
```suggestion
def generic_command(client: MsGraphClient, args: Dict[str, Any]) -> CommandResults:
```